tests/tear-down.pl

```diff
@@ -1,12 +0,0 @@
-use strict;
-
-my $fail = 0;
-
-system("dropdb -p 6433 hydra-test-suite") == 0 or $fail = 1;
-system("pg_ctl -D postgres -w stop") == 0 or $fail = 1;
-
-system("chmod -R a+w nix") == 0 or $fail = 1;
-system("rm -rf postgres data nix git-repo hg-repo svn-repo svn-checkout svn-checkout-repo bzr-repo bzr-checkout-repo darcs-repo") == 0 or $fail = 1;
```

Comment:
Looks like this line has been lost without replacement.

Reply:
This has been resolved in what would be a follow-up UP https://github.com/grahamc/hydra/pull/3 by creating these repositories in a per-test scratch directory.